examples/interesting_deps/WORKSPACE

```diff
@@ -38,6 +38,7 @@ load("@cgrindel_rules_spm//spm:defs.bzl", "spm_pkg", "spm_repositories")
 
 spm_repositories(
     name = "swift_pkgs",
+    build_mode = "bazel",
```

Comment:
Unfortunately, `bazel` build mode is [not quite ready for primetime](https://github.com/cgrindel/rules_spm/discussions/157).  Try removing this. Then, we can chase down any CI issues.